### DISCUSSION TITLE
"bypassDNA" equivalent within nested folders?

### DISCUSSION THREAD

**Discussion by j-greig** [nftchef/art-engine#279] [Help + Q&A]
I've switched from defining layers in the config to nesting them.

Previously I had an "Ears" layer that was bypassed and removed from the traits so that for all intents and purposes it was part of the "Head" layer below it.

Is there a way to achieve this within a nested folder structure — imagine you could prefix a layer so that it wasn't included in the metadata, but an image layer inside it was still picked and included at random in the image output....

**Comment by nftchef** [comment]
the easiest solution  to something like this when extra traits are output is to probably run the `remove trait` util _after generation_

https://github.com/nftchef/art-engine#remove-trait-util
